### DISCUSSION TITLE
Direkte Spannungsmessung vom Akku

### DISCUSSION THREAD

**Discussion by E-Ulli** [hoylabs/OpenDTU-OnBattery#261] [Ideas]
Da wir den Akku über einen Step-Up-Wandler betreiben, kann man über den WR nicht die Spannung Überwachen. Der Akku hat ein BMS über Bluetooth was man sicher nur schwer einbinden kann. Also wäre als universelle alternative eine Spannungsmessung über den ESP32 oder evtl. ein Canbus oder I²C Spannungsmess-IC schön.

**Comment by uli-bs** [comment]
Wo ist der Vorteil vor den HM einen Step-Up zu pflanzen?
IMO versaut das nur den Wirkungsgrad (besser als 95% wird der auch nicht sein, verschenkt also in Summe dann schon gut 10%)

**Reply by spcqike** [reply]
Man kann eine vorhandene alte 12V Batterie nehmen und braucht keine zweite (und dazu einen balancer) oder gar eine neue mit 24V oder 48V :)

**Reply by uli-bs** [reply]
OK, an das 12V Scenario habe ich nicht gedacht.
Es wird nur an anderen Stellen auch für 24V beschrieben/empfohlen, einen UP-Wandler einzusetzen, um den WR in den MPPT Bereich zu bringen, was aber nach meinen Beobachtungen, zumindest bei den HM WRs nicht wirklich nötig ist.

**Comment by E-Ulli** [comment]
Natürlich nur Nachteile. Aber wie spcqike schon schreibt wäre eine neue oder zweite Batterie zu teuer. Aber das ist nicht das Thema.

**Comment by E-Ulli** [comment]
1. MPPT sollte bei Batterie nie gemacht werden. 2. Der HM braucht sehr lange (Minuten) zum Einschalten wenn er nur 20-30V bekommt.

**Comment by E-Ulli** [comment]
Da die Spannungsüberwachung auch nur bei dem Regelkanal funktioniert kann man ihn auch nicht austricksen in dem man an den zweiten Eingang den 12V Akku direkt anschließt. Dabei ist mir ein werkwürdiges Verhalten aufgefallen. CH0: 49V dyn-Regelung aktiv auf diesem Kanal. CH1: 12V Hoymiles regelt CH1 auf 50 Watt hoch und versucht dann CH0 auch hochzuregeln bricht dann aber ab. Eigentlich dürfte bei 12V gar nichts passieren da die Einschaltschwelle >16V liegt.  Bei manueller Limit-Vorgabe passiert das auch nicht nur bei der Dynamischen Regelung. Irgendwas kann da nicht stimmen.

**Comment by Manos1966** [comment]
Hoymiles **Anlaufspannung (V)** ist 22V
Bevor 22V kommen, startet der Hoymiles nicht.
Danach, produziert er bis 16V (Betriebsspannungsbereich (V) 16 bis 60V )
![Hoymiles Spannungsbereich](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/b7202295-4fba-46ed-aa87-ab5ed7faac43)
